src/Tools/CoreFx.Tools/IsManagedAssembly.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.DotNet.Build.Tasks
+{ 
+    public partial class GenerateDepsJson
```

Comment:
Wasn't this split before because you had two implementations of `IsManagedAssembly` ? There's just one now, so it can all be in the same file.